package.json

```diff
@@ -50,13 +50,13 @@
     "@angular/compiler-cli": "^9.0.3",
     "@angular/language-service": "^9.0.3",
     "@testing-library/jest-dom": "^4.1.0",
-    "@types/jest": "~24.0.11",
+    "@types/jest": "~25.2.3",
     "@types/node": "^13.7.6",
     "codelyzer": "^5.1.2",
     "cpy-cli": "^3.1.0",
     "husky": "^2.3.0",
-    "jest": "^24.1.0",
-    "jest-preset-angular": "^7.1.1",
+    "jest": "^26.0.1",
+    "jest-preset-angular": "^8.2.0",
```

Comment:
Upgrade `jsdom` which includes some bugfixes for `selectOptions` (`blur` on a removed element)